config/locales/en.yml

```diff
@@ -285,8 +285,8 @@ en:
 
         location:
           deactivated:
-            startdate: "The location %{postcode} was deactivated on %{date} and was not available on the day you entered. Select another location or edit the tenancy start date"
-            location_id: "The location %{postcode} was deactivated on %{date} and was not available on the day you entered. Select another location or edit the tenancy start date"
+            startdate: "The location %{postcode} is inactive on this date. Enter another date or choose another location."
+            location_id: "This location is not active on the tenancy start date. Choose another location or edit the tenancy start date."
```

Comment:
Would we also want to remove `date` arguments from wherever we're using these error messages?

Reply:
Although the error message defined by `location.deactivated.location_id `no longer needs postcode or date, the other scopes do e.g. `location.activating_soon.location_id` so must still pass date.